src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -10,12 +10,26 @@
     public static final String COMMAND_WORD = "help";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
+            + "Example: " + COMMAND_WORD + " [COMMAND]";
```

Comment:
Maybe the example can be "COMMAND_WORD + " meet" ?
Cause it's an example and not the format? Unless you're going for format then i think better to replace "Example:" with "Format:" 😅

Reply:
Right good catch. 👍